### PR TITLE
Help about page

### PR DESCRIPTION
**Pull request by teryhill**
Added help about page

### CODE REVIEW

File: interface/main/about_page.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ </head>
+  <body class="body_top">
+    <div style="text-align: center;">
+    <span class="title"><?php  echo xlt('About LibreEHR'); ?> </span><br><br>  
+	<span class="text"><?php  echo xlt('Version Number'); ?>: <?php echo "v".text($openemr_version) ?></span><br><br>
+    <?php if (!empty($GLOBALS['support_phone_number'])) { ?>
+   	  <span class="text"><?php  echo xlt('Support Phone Number'); ?>: <?php echo $GLOBALS['support_phone_number'] ?></span><br><br>
+    <?php } ?>
+   </div>
+    <a href="<?php echo "http://open-emr.org/wiki/index.php/OpenEMR_".attr($v_major).".".attr($v_minor).".".attr($v_patch)."_Users_Guide"; ?>" target="_blank" class="css_button"><span><?php echo xlt('User Manual'); ?></span></a><br><br>
```
Comment (aethelwulffe) [inline interface/main/about_page.php]: URL change?

**Comment by aethelwulffe** [comment]
Code passes muster, and is relevant to Milestone1 under the frameless integration heading.
Recommend merging pull request.

**Comment by tmccormi** [comment]
@teryhill this pull request has conflicts now and must be rebased

**Comment by teryhill** [comment]
Do we have a URL for this to point to for the WIKI?

**Pull request merged**